eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -51,51 +51,94 @@ major_protocol_version = 1..next_major_protocol_version
 protocol_version = (major_protocol_version, uint)
 
 transaction_body =
-  { 0 : set<transaction_input>        ; inputs
+  { 0 : set<transaction_input>      ; inputs
   , 1 : [* transaction_output]
-  , 2 : coin                          ; fee
-  , ? 3 : uint                        ; time to live
+  , 2 : coin                        ; fee
+  , ? 3 : uint                      ; time to live
   , ? 4 : [* certificate]
   , ? 5 : withdrawals
   , ? 7 : auxiliary_data_hash
-  , ? 8 : uint                        ; validity interval start
+  , ? 8 : uint                      ; validity interval start
   , ? 9 : mint
   , ? 11 : script_data_hash
-  , ? 13 : set<transaction_input>     ; collateral inputs
+  , ? 13 : set<transaction_input>   ; collateral inputs
   , ? 14 : required_signers
   , ? 15 : network_id
-  , ? 16 : transaction_output         ; collateral return
-  , ? 17 : coin                       ; total collateral
-  , ? 18 : set<transaction_input>     ; reference inputs
-  , ? 19 : [* governance_action_info] ; New; governance actions
-  , ? 20 : [* vote]                   ; New; votes
+  , ? 16 : transaction_output       ; collateral return
+  , ? 17 : coin                     ; total collateral
+  , ? 18 : set<transaction_input>   ; reference inputs
+  , ? 19 : [* governance_procedure] ; New; governance procedures
   }
 
-governance_action_info =
-  [ deposit_amount    : coin
+governance_action_state =
```

Comment:
I might be forgetting something, why do we have the `governance_action_state` in the wire spec? it's not anywhere in a transaction, I don't think users will care about it...